src/Honeybee_ Run Energy Simulation.py

```diff
@@ -1373,7 +1385,7 @@ def main(north, epwFileAddress, EPParameters, analysisPeriod, HBZones, HBContext
     
     #Ground Temperatures.
     if grndTemps == []:
-        groundtemp = lb_preparation.groundTempData(_epwFile,[])
+        groundtemp = lb_preparation.groundTempData(_epwFile,[]) # XXX Print unwanted average temperatures at 3 different depths
```

Comment:
@chriswmackey  Chris I discovered a bug in the RunEnergyPlus component when it is calling the lb_preparation.groundtempData function because this function actually prints out the 3 different depths of the ground temperature data these depths are printed out in the run energy plus report. I think you wrote this code right? do you want me to create a work out and fix it or would you like to?
![ground temperature bug](https://cloud.githubusercontent.com/assets/8773483/9423246/e4d05698-48b3-11e5-827b-9a2d9a704f3a.PNG)